Assets/AY_Scripts/ParticlesService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ParticlesService : MonoSingletonGeneric<ParticlesService>
+{
+    [SerializeField] private ParticleSystem tankExplosion;
+    [SerializeField] private ParticleSystem shellExplosion;
+
+    public void GetTankExplosion(Transform tr)
+    {
+        ParticleSystem ps = Instantiate(tankExplosion, tr.position, tr.rotation);
+        ps.Play();
+        Destroy(ps.gameObject, 3f);
```

Comment:
Destroy the particle system after the particle system has finished playing. Find a way to get that duration.